draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -790,57 +793,55 @@ The transparency service maintains a mirror of the CA's latest batch number, and
+[[TODO: If the mirror gets far behind, or if the CA just stops publishing for a while, it may suddenly have to catch up on many batches. Should we allow the mirror to catch up to the latest validity window and skip the intervening batches? The intervening batches are guaranteed to have been expired #37 ]]
 
-If the relying party has a trusted update service, but the update service does not have the resources to mirror the full batch state, the transparency service can be composed of this update service and several, less trusted mirrors. In this model, the mirrors are not trusted to serve authoritative trust anchor information to relying parties, but the update service trusts at least half of them to faithfully and consistently mirror the batch state.
+## Relying Party Policy
 
-Each mirror follows the procedure in {{single-trusted-service}} to maintain and publish a mirror of the CA's batch state.
+In order to accept certificates from a CA that it trusts, a relying party must, in the background, obtain an up-to-date copy of the CA's validity window. In doing so, a relying party SHOULD ensure the following properties:
 
-The update server maintains the latest validity window validated to appear in all mirrors. It updates this by polling the mirrors and running the following steps:
+Authenticity:
+: The relying party only accepts validity windows that were certified by the CA
 
-1. For each mirror, fetch the latest batch number.
+Transparency:
+: The relying party only accepts validity windows covering certificates that are published in some publicly-accessible form, so that, in particular, the subject of the certificate can notice any unauthorized certificates
 
-2. Let `new_latest_batch` be the highest batch number that is bounded by the value fetched from at least half of the mirrors. Let `old_latest_batch` be the batch number of the currently stored validity window.
+How the relying party does this has no direct impact on certificate issuance ({{issuing-certificates}}) or usage ({{using}}). Different relying parties can obtain validity windows from different sources or apply different policies, while supporting the same unmodified certificates and CAs. Thus this section does not prescribe particular policies or mechanisms, but provides examples and general guidance. Relying parties MAY implement policies other than those described below, and MAY incorporate entities acting in roles not described in this document.
 
-3. If `new_latest_batch` equals `old_latest_batch`, finish this procedure without reporting an error.
+Relying parties SHOULD ensure transparency by obtaining validity windows from the CA and/or some combination of trusted mirrors. The relying party picks sources which, together, are trusted to satisfy the requirements described in {{mirrors}}. Mirrors allow the relying party to maintain transparency in the face of a misbehaving CA that may, for example, stop serving some unauthorized certificate in a batch to evade detection.
```

Comment:
I like *misbehaving or compromised*. Transparency isn't just about keeping CAs honest, but also giving a well-behaving CA some assurance they've not been compromised.

Reply:
Fair enough. I guess I would say that the CA is the service with a private key and the _CA operator_ is the entity that maintains it. If the CA is compromised, the _CA operator_ may well be honest, but the service is still misbehaving. But people are often sloppy enough about this that adding the extra word SGTM.

```suggestion
Relying parties SHOULD ensure transparency by obtaining validity windows from the CA and/or some combination of trusted mirrors. The relying party picks sources which, together, are trusted to satisfy the requirements described in {{mirrors}}. Mirrors allow the relying party to maintain transparency in the face of a misbehaving or compromised CA that may, for example, stop serving some unauthorized certificate in a batch to evade detection.
```